### PR TITLE
Add exists function to file api

### PR DESCRIPTION
**Pull request by amjadhifthikar**

### CODE REVIEW

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Exists.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.ballerinalang.natives.annotations.BallerinaAnnotation;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+import org.ballerinalang.natives.annotations.ReturnType;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Can be used to see whether a file exists.
```
Comment (chanakaudaya) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Exists.java]: Better to replace "see" with "check"?

File: modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java [hunk 2]
```diff
@@ -175,6 +176,19 @@ public void testMoveDir() throws IOException {
     }
 
     @Test
+    public void testExists() throws IOException {
+        String targetPath = "temp/exist-file.txt";
```
Comment (chanakaudaya) [inline modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java]: Can you extend this test to check the non-exists scenario also?

Comment (amjadhifthikar) [inline modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java]: Added it

Comment (chanakaudaya) [inline modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/functions/FileTest.java]: Thanks

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Exists.java (hunk 1 above)
Comment (amjadhifthikar) [reply modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Exists.java]: Fixed with 52cfa08

**Pull request merged**